### PR TITLE
homework_6_done

### PR DESCRIPTION
**Pull request by Nataly1405**
Added HomeWork_3  - Nataly

### CODE REVIEW

File: Nataly_Bondarenko/HW6/task1_.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+for element in range(0, 100, 1):
+    left_operand = randint(0, 100)
+    right_operand = randint(0, 100)
+    operator = randint(1, 3)
+    list_of_tuple.append((left_operand, operator, right_operand))
+
+# print(type(list_of_tuple))
+
+import os
```
Comment (bohdandovbsyh) [inline Nataly_Bondarenko/HW6/task1_.py]: all imports should be in one place at the beginning of file

File: Nataly_Bondarenko/HW6/task1_.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+try:
+    os.makedirs('test/data')
+except FileExistsError:
+    print("ERROR file exists: 'test/data'")
```
Comment (bohdandovbsyh) [inline Nataly_Bondarenko/HW6/task1_.py]: it is folders not files

File: Nataly_Bondarenko/HW6/task3_count_letters.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Please count how many times each letter appears in this file
+from collections import Counter
+
+with open('text.txt', 'r') as file:
+    data = file.read()
+
+rep_list = ['-', '.', ' ', "'", "[", ":", '"', ']', 'â€', '”', '3', '2', ';',
+            '(', ')', '\n', '7', '0', ',']
```
Comment (bohdandovbsyh) [inline Nataly_Bondarenko/HW6/task3_count_letters.py]: what if the text will have some char that is not in this list?

Mostly this solution is okay but only for this text, but it is better to remove all charm which does not a letter using regex

File: Nataly_Bondarenko/HW6/task1_.py (hunk 2 above)
Comment (Nataly1405) [reply Nataly_Bondarenko/HW6/task1_.py]: fixed

File: Nataly_Bondarenko/HW6/task3_count_letters.py (hunk 3 above)
Comment (Nataly1405) [reply Nataly_Bondarenko/HW6/task3_count_letters.py]: fixed

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**